### PR TITLE
[MM-14720] Add bot post counts per day to System Analytics 

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR adds a new component to the System Console Site Statistics page.  

The new chart is located directly below the `Total Posts` chart.
image

#### Jira Ticket Link
https://mattermost.atlassian.net/browse/MM-14720

#### Related Pull Requests
- Has server changes (https://github.com/mattermost/mattermost-server/pull/11402)
- Has redux changes (https://github.com/mattermost/mattermost-redux/pull/868)
- Has mobile changes (N/A)

### CODE REVIEW

**Comment by jfrerich** [comment]
@crspeller, @lieut-data How do I resolve the untagged-build required check?

File: components/analytics/team_analytics/team_analytics.jsx [hunk 1]
```diff
@@ -130,6 +131,7 @@ export default class TeamAnalytics extends React.Component {
 
         const stats = this.props.stats[this.state.team.id];
         const postCountsDay = formatPostsPerDayData(stats[StatTypes.POST_PER_DAY]);
+        const botPostCountsDay = formatPostsPerDayData(stats[StatTypes.BOT_POST_PER_DAY]);
```
Comment (crspeller) [inline components/analytics/team_analytics/team_analytics.jsx]: This is the problem. This isn't used... should it be?

**Comment by crspeller** [comment]
@jfrerich Looks like you error is:
```
134:15  error  'botPostCountsDay' is assigned a value but never used  no-unused-vars
```

You can click the "details" to find the error.

**Comment by jfrerich** [comment]
@crspeller 	All three repos are now clean for merging.  I also ran into an error with `i18n/en.json` so I added the offending line in that file.  (see last commit)

Thanks for the reviews.

**Comment by crspeller** [comment]
@jfrerich Can you update the redux commit for this? (since there are redux changes)

**Comment by jfrerich** [comment]
> @jfrerich Can you update the redux commit for this? (since there are redux changes)

Hey @crspeller, what are you asking for?  What else was required, in addition to the redux PR https://github.com/mattermost/mattermost-redux/pull/868?

**Comment by crspeller** [comment]
@jfrerich You have to update the redux commit in the `package.json` file to point to your merged changes. Otherwise it won't pick up the changes. Likely someone else has set the commit beyond your changes, so merging master would be enough.

**Comment by jfrerich** [comment]
@crspeller Thanks for the explanation.  The webapp `package.json` currently points to `mattermost-redux` commit that is 4 commits newer than what we merged.  I merged the latest master in this branch and now we are clean.

image

image

**Comment by aaronrothschild** [comment]
@jfrerich I'm updating the docs to include the new chart you added.  Can you verify that the difference between these two charts is correct (subtle):

`Total Posts (graph)
    The total number of posts made on a certain day in all the teams on your system, including deleted posts and posts made using automation.`
    
New Chart `Total Posts from Bots (graph)
    The total number of posts made on a certain day in all the teams on your system, including deleted posts, exclusively made using automation.`

https://github.com/mattermost/docs/pull/2898

**Comment by jfrerich** [comment]
Hi, @aaronrothschild.  FYI, I haven't forgotten about you and was working on another 5.14 ticket.  I'm 99% these charts are correct and the calculations match as you described.  If we are going to add charts, I would recommend creating two that don't match so closely.   The reason the two above are so similar is that I took a portion of the users in the sampledata and converted them to bots.  I can generate some better looking (more realistic) charts if you'd like.

**Pull request merged**